### PR TITLE
Adding AlertsSuppressionRule cmdlets to Az.Security

### PR DESCRIPTION
**Pull request by nitsi**
## Description

Adding new capabilities for alerts suppression rule - user can create, delete, update and get rules (create and update are both the same "Set").

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - [design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1168)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Security/Security/Az.Security.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.1.1'
+ModuleVersion = '1.1.2'
```
Comment (isra-fel) [inline src/Security/Security/Az.Security.psd1:15]: Do not update the version - it's calculated during release process

**Review by isra-fel — CHANGES_REQUESTED** [review]
Please resolve the conflicts

File: src/Security/Security/Az.Security.psd1 (hunk 1 above)
Comment (nitsi) [reply src/Security/Security/Az.Security.psd1]: Reverted

**Comment by nitsi** [comment]
@isra-fel I'm getting the following error:
"Az.Security","Microsoft.Azure.Commands.Security.Cmdlets.AlertsSuppressionRules.NewAlertsSuppressionRuleScope","New-AzAlertsSuppressionRuleScope","1","8100","New-AzAlertsSuppressionRuleScope Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
  
NewAlertsSuppressionRuleScope has `SupportsShouldProcess = false` do I need to support this?

**Comment by isra-fel** [comment]
@nitsi no, in-memory cmdlets do not support ShouldProcess.
CI failed simply because the rule was set to detect by cmdlet name (verb).
Please follow this guide to suppress the error: https://github.com/Azure/azure-powershell/blob/main/documentation/Debugging-StaticAnalysis-Errors.md#breaking-changes

**Comment by nitsi** [comment]
@isra-fel tests and static analysis passed, "azure-powershell - security-tools" have not started yet (2h), is there a way to restart it?

**Review by isra-fel — APPROVED** [review]

**Pull request merged**